corehq/apps/domain/templates/domain/admin/info_base.html

```diff
@@ -1,13 +0,0 @@
-{% extends "hqwebapp/base_section.html" %}
```

Comment:
Deleted this altogether because most of its descendants don't actually have any autocomplete_fields, and those that do can be removed.